02_configure_host.sh

```diff
@@ -306,6 +306,8 @@ if [ "$MANAGE_BR_BRIDGE" == "y" ] ; then
     sudo virsh net-destroy ${BAREMETAL_NETWORK_NAME}
     sudo nmcli con del ${BAREMETAL_NETWORK_NAME}
     sudo virsh net-start ${BAREMETAL_NETWORK_NAME}
+    # Needed in IPv6 on some EL9 hosts for the bootstrap VM to get an IP
+    echo 0 | sudo dd of=/proc/sys/net/ipv6/conf/${BAREMETAL_NETWORK_NAME}/addr_gen_mode
```

Comment:
@cybertron @mkowalski without this my bootstrap VM doesn't get any IP. Any idea why or if there is a better way I could apply it(I couldn't find a way to get it to work with nmcli)?

Reply:
My only concern is that echo-ing into /proc/sys is very temporary and NetworkManager will revert that if at any point in time BAREMETAL_NETWORK_NAME goes down&up again. It may not be a real concern, but something to keep in mind.

Would this nmcli call work for you?
```
nmcli conn modify ${BAREMETAL_NETWORK_NAME} ipv6.addr-gen-mode 0
```

The ultimately persistent method is

```
/etc/NetworkManager/NetworkManager.conf
[...]
[connection]
ipv6.addr-gen-mode=0
```
to have it as a global default - that's what we do in real OCP clusters - https://github.com/openshift/machine-config-operator/blob/master/templates/common/openstack/files/ipv6-config.yaml

Reply:
P.S. Be aware of a bug in older NetworkManager related to this feature - https://access.redhat.com/solutions/7016090. Again, it may be not relevant, but it's so subtle that at some point may be helpful to know about it

Reply:
I tried "nmcli conn modify ${BAREMETAL_NETWORK_NAME} ipv6.addr-gen-mode 0" but it didn't work 

I didn't try updating "/etc/NetworkManager/NetworkManager.conf" 

how would you feel about merging this as is and I'll circle back and try updating NetworkManager.conf once the CI nodes are all on EL9?

Reply:
Right, indeed something is fishy with mode `0`...

```
[root@localhost ~]# nmcli conn show eno1 | grep addr-gen-mode
ipv6.addr-gen-mode:                     eui64
[root@localhost ~]# nmcli conn modify eno1 ipv6.addr-gen-mode 0
[root@localhost ~]# nmcli conn show eno1 | grep addr-gen-mode
ipv6.addr-gen-mode:                     eui64
[root@localhost ~]# nmcli conn modify eno1 ipv6.addr-gen-mode 1
[root@localhost ~]# nmcli conn show eno1 | grep addr-gen-mod
ipv6.addr-gen-mode:                     stable-privacy
[root@localhost ~]# nmcli conn modify eno1 ipv6.addr-gen-mode 2
[root@localhost ~]# nmcli conn show eno1 | grep addr-gen-mod
ipv6.addr-gen-mode:                     default-or-eui64
[root@localhost ~]# nmcli conn modify eno1 ipv6.addr-gen-mode eui64
[root@localhost ~]# nmcli conn show eno1 | grep addr-gen-mod
ipv6.addr-gen-mode:                     eui64
```

Go ahead with merging, your solution is perfectly valid one